meson.options

```diff
@@ -25,6 +25,7 @@ option('integration_tests', type: 'boolean',
 option('namespace', type: 'string',
        description: 'A prefix for exported symbols')
 option('device', type: 'boolean', description: 'Build device libraries', value: false)
+option('testing', type: 'boolean', description: 'Build testing libraries', value: false)
```

Comment:
Is there any advantage to adding this as a new build option versus just having it tied to the existing `tests` option?